examples/Extensions/AddHotelCallout.php

```diff
@@ -0,0 +1,321 @@
+            $customerId,
+            [$extensionFeedItemOperation]
+        );
+        $extensionFeedItemResourceName = $response->getResults()[0]->getResourceName();
+        printf(
+            "Created an extension feed item with resource name: '%s'.%s",
+            $extensionFeedItemResourceName,
+            PHP_EOL
+        );
+
+        return $extensionFeedItemResourceName;
+    }
+
+    /**
+     * Adds the extension feed item to the customer account.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the client customer ID
+     * @param string $extensionFeedItemResourceName the extension feed item resource name
+     */
+    private static function addExtensionToAccount(
+        GoogleAdsClient $googleAdsClient,
+        int $customerId,
+        string $extensionFeedItemResourceName
+    ): void {
+        // Creates a customer extension setting, sets its type to HOTEL_CALLOUT, and attaches the
+        // feed item.
+        $customerExtensionSetting = new CustomerExtensionSetting([
+            'extension_type' => ExtensionType::HOTEL_CALLOUT,
+            'extension_feed_items' => [new StringValue(['value' => $extensionFeedItemResourceName])]
+        ]);
+        // Creates a customer extension setting operation.
+        $customerExtensionSettingOperation = new CustomerExtensionSettingOperation();
+        $customerExtensionSettingOperation->setCreate($customerExtensionSetting);
+
+        // Issues a mutate request to add the customer extension setting and print its information.
+        $customerExtensionSettingServiceClient =
+            $googleAdsClient->getCustomerExtensionSettingServiceClient();
+        $response = $customerExtensionSettingServiceClient->mutateCustomerExtensionSettings(
+            $customerId,
+            [$customerExtensionSettingOperation]
+        );
+        printf(
+            "Created a customer extension setting with resource name: '%s'.%s",
+            $response->getResults()[0]->getResourceName(),
+            PHP_EOL
+        );
+    }
+
+    /**
+     * Adds the extension feed item to the specified campaign.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the client customer ID
+     * @param int $campaignId the campaign ID
+     * @param string $extensionFeedItemResourceName the extension feed item resource name
+     */
+    private static function addExtensionToCampaign(
+        GoogleAdsClient $googleAdsClient,
+        int $customerId,
+        int $campaignId,
+        string $extensionFeedItemResourceName
+    ): void {
+        // Creates a campaign extension setting, sets its type to HOTEL_CALLOUT, and attaches the
+        // feed item.
+        $campaignExtensionSetting = new CampaignExtensionSetting([
+            'campaign' => new StringValue(
+                ['value' => ResourceNames::forCampaign($customerId, $campaignId)]
```

Comment:
better to have the `[` in previous line and `]` in next line

Reply:
To me, both are equally fine and PSR-12 compliant. I modified as requested though.